mne/evoked.py

```diff
@@ -526,6 +526,12 @@ def plot_topomap(self, times=None, ch_type='mag', layout=None, vmin=None,
             starts 5 ms before and ends 5 ms after a given time point.
             Defaults to None, which means no averaging.
         """
+        if format is not None:
```

Comment:
no, but you didn't introduce the new argument `fmt`. You need to introduce it and then deprecate `format`